.github/workflows/ci.yml

```diff
@@ -123,6 +123,72 @@ jobs:
       - run: make man
         working-directory: src/github.com/containerd/containerd
 
+  # Make sure binaries compile with other platforms
+  crossbuild:
+    name: Crossbuild Binaries
+    needs: [project, linters, protos, man]
+    runs-on: ubuntu-18.04
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - goos: linux
+            goarch: arm64
+          - goos: linux
+            goarch: arm
+            goarm: "7"
```

Comment:
Is 7 the lowest version of 32-bit Arm we want to support?

Reply:
We could add more and maybe worthwhile since I know folks use v5 and v6 with Docker.

Reply:
I'll also add windows+armv7 but this is known broken right now.

Reply:
@alexellis might have opinion on how "low" to go with arm 32-bit based on his connection to that community

Reply:
I'd suggest 6 is worth adding (that's Pi1 and Pi0, which are fairly popular), but even I'd consider 5 to be "best effort" (and not worth the CPU time to test explicitly).

(my 2¢)

Reply:
Oh, though cgo means armv6 is not something you'll be able to easily cross-compile from an Ubuntu environment unless you build/download a cross toolchain from elsewhere, so the most you could do is also add 5, which should cover the bases (if the 7 build works and a 5 build works, the odds of 6 having problems are really low -- they'd be toolchain mistakes, not code problems, at that point, like using the wrong C compiler :smile:).

Reply:
And got the windows/arm/v7 build fixed as well (see go-winio update), so added that in.

Reply:
Also added armv5, guess I didn't comment on that.